utils/json.go

```diff
@@ -0,0 +1,103 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type NvdCVEs struct {
+	Entries []NvdCVEEntry `json:"CVE_Items"`
+}
+
+type NvdCVEEntry struct {
+	CVE               NvdCVE    `json:"cve"`
+	Impact            nvdImpact `json:"impact"`
+	PublishedDateTime string    `json:"publishedDate"`
```

Comment:
_Should_ this be?:

```suggestion
	PublishedDateTime string    `json:"published_date"`
```

Reply:
Nah, this field is `publishedDate` in the marshalled json.